library/core/src/slice/mod.rs

```diff
@@ -640,6 +640,11 @@ impl<T> [T] {
     /// Calling this method with an out-of-bounds index is *[undefined behavior]*
     /// even if the resulting reference is not used.
     ///
+    /// You can think of this like `.get(index).unwrap_unchecked()`.  It's UB
+    /// to call `.get_unchecked(len)`, even if you immediately convert to a
+    /// pointer.  And it's UB to call `.get_unchecked(..len +1)`,
```

Comment:
It should probably be either `.get_unchecked(..len + 1)` or `.get_unchecked(..len+1)` (notice the spaces).

Reply:
rustfmt formats as `(..len + 1)`, that should be used for consistency.